hongyujin/week04/Baekjoon_10797.py

```diff
@@ -0,0 +1,12 @@
+import sys
+input = sys.stdin.readline
+
+day = int(input())
+car = list(map(int, input().split()))
+count = 0  # 위반하는 차량 수
+
+for i in range(len(car)):
```

Comment:
```suggestion
count = car.count(day)
```
for 문 대신에 count 함수를 쓰면 같은 숫자 찾기 더 쉬울거에요..!!